### DISCUSSION TITLE
Trigger modal with text link?

### DISCUSSION THREAD

**Discussion by ghost** [Automattic/wordpress-activitypub#672] [Q&A] [answered]
Hi! Firstly, thanks for the great plugin. It's got me really excited about the possibilities of ActivityPub and the fediverse.

Secondly, apologise if this is a silly question. I'm wondering if there's a way to trigger the modal with instructions on how to follow that pops up when you click the "Follow" button in the "Follow me on the Fediverse" block.

I like the modal. But the block as such does not necessarily fit everywhere, stylistically speaking.

Thanks!

**Comment by mattwiebe** [comment, marked as answer]
We'd like to make the Follow button something that can be standalone, too. Thanks!

**Comment by ghost** [comment]
Thanks, looking forward to it! You can see [here](https://lars-christian.com/all-posts/) an example of what I'd like to do. I'm relying on a third party to facilitate following through simple links as of right now. Bringing up the plugin's modal would be much smoother.
